toolkit/resources/manifests/package/toolchain_aarch64.txt

```diff
@@ -96,17 +97,17 @@ gettext-0.21-2.cm2.aarch64.rpm
 gettext-debuginfo-0.21-2.cm2.aarch64.rpm
 gfortran-11.2.0-2.cm2.aarch64.rpm
 glib-2.71.0-1.cm2.aarch64.rpm
-glib-debuginfo-2.71.0-1.cm2.aarch64.rpm
-glib-devel-2.71.0-1.cm2.aarch64.rpm
-glib-doc-2.71.0-1.cm2.noarch.rpm
-glib-schemas-2.71.0-1.cm2.aarch64.rpm
 glibc-2.34-3.cm2.aarch64.rpm
 glibc-devel-2.34-3.cm2.aarch64.rpm
 glibc-i18n-2.34-3.cm2.aarch64.rpm
 glibc-iconv-2.34-3.cm2.aarch64.rpm
 glibc-lang-2.34-3.cm2.aarch64.rpm
 glibc-nscd-2.34-3.cm2.aarch64.rpm
 glibc-tools-2.34-3.cm2.aarch64.rpm
+glib-debuginfo-2.71.0-1.cm2.aarch64.rpm
+glib-devel-2.71.0-1.cm2.aarch64.rpm
+glib-doc-2.71.0-1.cm2.noarch.rpm
+glib-schemas-2.71.0-1.cm2.aarch64.rpm
```

Comment:
It's the output of the `sort` command, which is needed before passing stuff to the `comm` tool.  As far as I can tell its based on the locale. Unless you force it to ascii mode it will ignore `-` and similar characters when sorting. I've added a --check-order to the comm tool which will double check things sort as its expecting.

Reply:
In that case, I'd still preserve the previous order - it keeps the packages with the same prefix together and I feel it's more common for a human being in general. We can still use `sort` before the diff, though.

Reply:
If we get back to use the update_manifest script its going to overwrite that order though, and I think we need to get into the habit of using it (once its working for 2.0 properly). I tried a few options to control the sort tool but didn't have much luck. I'll have another go at it.

Reply:
With `LC_COLLATE=C` and `-f` (ignore case) we get a sane sort order. I've left the sorts in the actual Makefile check alone since they just need to be internally consistent.